This provides the option to set the indicator for a NFS profile to true.
It may be necessary for users on NFS to use bookmarks/logging.
Will not be sending upstream.

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1428169

diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -1075,6 +1075,9 @@
 // allow META refresh by default
 pref("accessibility.blockautorefresh", false);
 
+// Whether profile is on NFS
+pref("storage.nfs_filesystem", true);
+            
 // Whether history is enabled or not.
 pref("places.history.enabled", true);
 
